code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1482,6 +1482,32 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 		target.grabbedby(user)
 		return 1
 
+/datum/species/proc/get_minimum_punch_damage(mob/living/carbon/human/user)
+	var/damage = punchdamagelow
+	if(HAS_TRAIT(user, TRAIT_IRONFIST))
+		damage += 3
+	if(HAS_TRAIT(user, TRAIT_PERFECT_ATTACKER)) // unit test no-miss trait
+		damage = get_maximum_punch_damage(user)
+	return damage
+
+/datum/species/proc/get_maximum_punch_damage(mob/living/carbon/human/user)
+	var/damage = punchdamagehigh
+	if(HAS_TRAIT(user, TRAIT_IRONFIST))
+		damage += 1
```

Comment:
```suggestion
		damage += 5
```
And fifteen maximum,

So Iron Fist is worth having outside of a very uncommon arena fight